packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -347,7 +405,7 @@ export class OpenId4VciHolderService {
     } as OpenId4VciTokenRequestOptions
 
     const tokenResponse = options.accessToken
-      ? { access_token: options.accessToken, c_nonce: options.cNonce }
+      ? { access_token: options.accessToken, c_nonce: options.cNonce, dPoPJwk: options.dPoPJwk }
       : await this.requestAccessToken(agentContext, tokenRequestOptions)
```

Comment:
For the service we can already change it (only api we need to worry about breaking changes)